### ISSUE TITLE
Improve SEO score by improving contrast ratio of background and foreground colors

### ISSUE THREAD

**Issue by hredi** [CMU-313/Teedy#12]
The current SEO score is 80. I would like to target the warning that states that the contrast ratio of background/foreground colors is insufficient, making it difficult for users to read

**Comment by hredi** [comment]
I would like to work on this please!
